sdks/python/apache_beam/runners/portability/fn_api_runner/fn_runner.py

```diff
@@ -426,7 +423,8 @@ def _collect_written_timers_and_add_to_deferred_inputs(
       pipeline_components,  # type: beam_runner_api_pb2.Components
       stage,  # type: translations.Stage
       bundle_context_manager,  # type: execution.BundleContextManager
-      deferred_inputs  # type: MutableMapping[str, PartitionableBuffer]
+      deferred_inputs,  # type: MutableMapping[str, PartitionableBuffer]
+      data_channel_coders,
```

Comment:
If possible try and keep the typing information on methods.

Reply:
Thanks for calling me out on this. The types themselves aren't very informative as to their meaning, but I added it here and elsewhere.